skynet/modules/ttt/summaries/jobs.py

```diff
@@ -71,7 +71,7 @@ async def create_job(job_type: JobType, payload: DocumentPayload, metadata: Docu
 
     await db.set(job_id, Job.model_dump_json(job))
 
-    log.info(f"Created job {job.id}.")
+    log.info(f"Created job {job}.")
```

Comment:
Unrelated change? What does this log now?

Reply:
'temp logging' commit, ill remove it soon